plugins/polio/api/perfomance_dashboard/serializers.py

```diff
@@ -0,0 +1,153 @@
+import logging
+
+from rest_framework import serializers
+
+from iaso.api.common import TimestampField
+from iaso.models import OrgUnit, User
+from plugins.polio.models.performance_dashboard import PerformanceDashboard
+
+logger = logging.getLogger(__name__)
+
+
+class UserNestedSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = User
+        fields = ["id", "username", "first_name", "last_name"]
+        ref_name = "UserNestedSerializerForNationalLogisticsPlan"
+
+
+class OrgUnitNestedSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = OrgUnit
+        fields = ["id", "name"]
+        ref_name = "OrgUnitNestedSerializerForNationalLogisticsPlan"
+
+
+class PerformanceDashboardListSerializer(serializers.ModelSerializer):
+    created_by = UserNestedSerializer(read_only=True)
+    updated_by = UserNestedSerializer(read_only=True)
+    created_at = TimestampField(read_only=True)
+    updated_at = TimestampField(read_only=True)
+
+    # For read operations, we want to display the country's name
+    country = OrgUnitNestedSerializer(read_only=True)
```

Comment:
```suggestion
    country_name = CharField(source="country.name", read_only=True)
```

Since we already have a field for `country_id` it may be easier to skip the serializer